pages/trial.tsx

```diff
@@ -0,0 +1,372 @@
+                      ) : (
+                        <div className="text-break my-1 h5">
+                          {" "}
+                          <span>
+                            {subdomain}
+                            <span className="text-nowrap">.grouparoo.app</span>
+                          </span>
+                        </div>
+                      )}
+                      <Error message={error.subdomain} />
+
+                      {errors.subdomain && (
+                        <small style={{ color: "red" }}>
+                          A valid subdomain is required.
+                        </small>
+                      )}
+
+                      <Button variant="link" onClick={onEditSubdomainClick}>
+                        {editSubdomain ? "save" : "change"}
+                      </Button>
+                    </div>
+                  </Form.Group>
+                  <Form.Group>
+                    <div className="my-3">
+                      <Form.Check
+                        {...register("check", { required: true })}
+                        type="checkbox"
+                        id="privacy-policy-check"
+                        defaultValue="false"
+                        label={privacyPolicyLabel}
+                      />
+                    </div>
+                  </Form.Group>
+                  <Form.Group className="text-center">
+                    <Button
+                      variant="primary"
+                      type="submit"
+                      className="btn-lg"
+                      disabled={disabled}
+                    >
+                      Submit
+                    </Button>
+                  </Form.Group>
+                </Form>
+              </Container>
+            ) : (
+              <>
+                <Container className="bg-white border shadow border-3 rounded col-11 p-4 mt-4 mt-md-0">
+                  <Row className="text-center mb-1">
+                    <Col>
+                      <h1>Grouparoo Cloud</h1>
+                      <h2>We'll be in touch!</h2>
+                    </Col>
+                  </Row>
+                  <Row>
+                    <Col>
+                      Your request for a trial of Grouparoo Cloud was
+                      successful. You will receive an email with your login
+                      credentials and instructions on how to get started
+                      shortly. Meanwhile you can check out:
+                    </Col>
+                  </Row>
+                  <div className="mt-3 mb-5">
+                    <Row className="my-3">
+                      <Col>
+                        <p className="h4">Getting Started Video</p>
+                        <a
+                          href="https://www.youtube.com/embed/kQ789gMXJB8?rel=0"
+                          target="_blank"
+                        >
+                          A quick intro into the Grouparoo platform
+                        </a>
```

Comment:
I'd suggest taking the user to `/docs/getting-started` instead - it's got the video and more info